src/components/number-picker/number-picker.component.ts

```diff
@@ -102,11 +102,21 @@ export class NumberPickerComponent implements ControlValueAccessor {
     /** Store the current valid state */
     _valid: boolean = true;
 
+    /**To determine if the component has been destroyed to avoid change detection being after the component
+     * is no longer instantiated. A workaround for Angular Forms bug (https://portal.digitalsafe.net/browse/EL-369) */
```

Comment:
I'm having trouble understanding this comment. Would it be fair to say that this is a flag to indicate when the component has been destroyed? Also the URL https://portal.digitalsafe.net/browse/EL-369 has nothing to do with Angular Forms. Is there an issue on the Angular GitHub that we can link to here?